### PR TITLE
docs: write down configuration file instructions

### LINKED ISSUE #67: No need of a configuration file (angular-cli-ghpages.json)
You have in your list of TODO's a configuration file to do not write all those parameters...

image

I think there is no necessary because the `options` property in the `angular.json` already has that feature and it works...

Take a look

image

And the dry-run says:

image

---------

The feature is great but could be donde using something that is already built. What about to write this down in the documentation ?

### PR DESCRIPTION
**Pull request by dianjuar**
Write down the instructions to put all command line options into the configuration file

Here you can have a [preview](https://github.com/bikecoders/angular-cli-ghpages/tree/67-write-configuration-file-instructions)

This close #67

### CODE REVIEW

**Comment by fmalcher** [comment]
> I'm wonder if we can provide a schema.json for those options all have all the benefits (auto-completion, validation, descriptions) 🤔

Yeah, that is something I also thought of these days, too. Can you open an issue? 😇 Thanks!

**Comment by JohannesHoppe** [comment]
In theory, this should already work:
 https://github.com/angular-schule/angular-cli-ghpages/blob/eb782e57a838a69b96bacdb1968a3be98683460b/src/builders.json#L6

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 ![Screenshot](docs/angular-cli-ghpages-deploy.gif)
 
-**Table of contents:**
```
Comment (JohannesHoppe) [inline README.md]: You should install https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig next time! ;-)

**Comment by JohannesHoppe** [comment]
@dianjuar I thank you very much for you effort. 
This is a very nice description. 🙂👍 

I will bake a new package very soon together with a new option, as discussed here:
https://github.com/angular-schule/ngx-deploy-starter/issues/1

**Comment by JohannesHoppe** [comment]
The updated documentation is included in
https://github.com/angular-schule/angular-cli-ghpages/releases/tag/v0.6.0 🚀

**Pull request merged**